packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout-selectors.js

```diff
@@ -4,6 +4,8 @@ export const ORDER_TIME_DROPDOWN = '[data-test-id="formfield-order-time-dropdown
 export const ORDER_TIME_DROPDOWN_OPTIONS = '[data-test-id="formfield-order-time-dropdown-select"] option';
 export const USER_NOTE_INPUT = '[data-test-id="user-note"] textarea';
 export const GO_TO_PAYMENT_BUTTON = '[data-test-id="confirm-payment-submit-button"]';
+export const KNOB_BUTTON = '[id$="tabbutton-knobs"]';
```

Comment:
Nit: This should eventually be put in a `shared-selectors` file. I imagine it'll be the same for all components.